include/ui/output-component.hpp

```diff
@@ -20,9 +20,12 @@
 #define ERAGPSIM_UI_OUTPUTCOMPONENT_HPP
 
 #include <QDebug>
+#include <QImage>
 #include <QQmlContext>
 #include "core/memory-access.hpp"
 
+// class QImage;
```

Comment:
I have no Idea what this does, I can't remember writing a newImage method in the output-Component, the only thing I remember writing is the getMemoryAccess method. Also a quick search does not yield any use for the newImage method